src/Controller.php

```diff
@@ -59,15 +59,17 @@ public function sync(OutputInterface $output, GlobalConfig $config, SyncOptions
     /**
      * @return void
      */
-    public function testSshConnection(OutputInterface $output, GlobalConfig $config, SyncOptions $options)
+    public function checkSshConnection(OutputInterface $output, GlobalConfig $config, SyncOptions $options)
```

Comment:
Does this also affect the actual command?

Reply:
No